scripts/build-types.ts

```diff
@@ -0,0 +1,24 @@
+import cue from 'cuelang-js';
+import { default as openapiTS } from 'openapi-typescript';
```

Comment:
`import` without `{}` is default. `import openapiTS from 'openapi-typescript';` I would call it `openApiToTS`

Reply:
Fixed in latest commit